cpp/ql/src/Security/CWE/CWE-020/IRExternalAPIsUsedWithUntrustedData.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name Frequency counts for external APIs that are used with untrusted data
```

Comment:
From the file names of these four queries, I find it difficult to distinguish between the ones that produce counts and the ones that produce "alerts". Can the file names be more like the `@id`s, starting with `Count`?

Reply:
I share your pain! I kept the names similar to Java, but seeing as I'm not the only one having this difficulty I renamed it in dea16d4d62ae529bd739c06d3123dd88fa5891f2.